philadelphia-perf-test/src/main/java/org/jvirtanen/philadelphia/perf/FIXMessageBenchmark.java

```diff
@@ -1,14 +1,28 @@
 package org.jvirtanen.philadelphia.perf;
 
-import static org.jvirtanen.philadelphia.fix42.FIX42Enumerations.*;
-import static org.jvirtanen.philadelphia.fix42.FIX42MsgTypes.*;
-import static org.jvirtanen.philadelphia.fix42.FIX42Tags.*;
+import static org.jvirtanen.philadelphia.fix42.FIX42MsgTypes.OrderSingle;
+import static org.jvirtanen.philadelphia.fix42.FIX42Tags.ClOrdID;
+import static org.jvirtanen.philadelphia.fix42.FIX42Tags.HandlInst;
+import static org.jvirtanen.philadelphia.fix42.FIX42Tags.MsgSeqNum;
+import static org.jvirtanen.philadelphia.fix42.FIX42Tags.MsgType;
+import static org.jvirtanen.philadelphia.fix42.FIX42Tags.OrdType;
+import static org.jvirtanen.philadelphia.fix42.FIX42Tags.Price;
+import static org.jvirtanen.philadelphia.fix42.FIX42Tags.SenderCompID;
+import static org.jvirtanen.philadelphia.fix42.FIX42Tags.SendingTime;
+import static org.jvirtanen.philadelphia.fix42.FIX42Tags.Side;
+import static org.jvirtanen.philadelphia.fix42.FIX42Tags.Symbol;
+import static org.jvirtanen.philadelphia.fix42.FIX42Tags.TargetCompID;
+import static org.jvirtanen.philadelphia.fix42.FIX42Tags.TransactTime;
```

Comment:
I think I would prefer to keep the imports from the enumeration, tag, and message type containers as they were for uniformity. Otherwise especially the number of import statements for tags in particular can get very high.